fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -1629,11 +1629,15 @@ relationPrimary
     | '(' VALUES rowConstructor (',' rowConstructor)* ')'
         (AS? alias=identifier columnAliases?)?                                          #inlineTable
     | subquery (AS? alias=identifier columnAliases?)?                                   #subqueryWithAlias
-    | qualifiedName '(' expressionList ')'
-        (AS? alias=identifier columnAliases?)?                                          #tableFunction
+    | tableFunctionCall (AS? alias=identifier columnAliases?)?                          #tableFunction
+    | TABLE '(' tableFunctionCall ')' (AS? alias=identifier columnAliases?)?            #tableFunctionTable
     | '(' relations ')'                                                                 #parenthesizedRelation
     ;
 
+tableFunctionCall
```

Comment:
I do not recommend adding a tableFunctionCall. Although this logic is not wrong. However, tableFunctionCall and FunctionCallExpr are not in one-to-one correspondence, which can easily cause trouble for other developers. I think it is enough to write directly, TABLE '(' qualifiedName '(' expressionList ')' ')' (AS? alias=identifier columnAliases?)?

Reply:
ditto